src/writeData/components/WriteDataDetailsContext.tsx

```diff
@@ -51,10 +51,15 @@ const WriteDataDetailsProvider: FC = ({children}) => {
     getAll<Authorization>(state, ResourceType.Authorizations)
   )
   const org = useSelector(getOrg)
+  const bucketname = useSelector(
+    (state: AppState) => state.dataLoading?.steps?.bucket
+  )
   const server = window.location.origin
 
+  const toLoadBucket = buckets?.find(b => b.name === bucketname)
+
   const [bucket, setBucket] = useState(
-    buckets[0] ?? ({name: '<BUCKET>'} as Bucket)
+    toLoadBucket ?? buckets[0] ?? ({name: '<BUCKET>'} as Bucket)
```

Comment:
A little fuzzy on the consequences of this with the other source of truth for bucket selections.

Reply:
the "buckets[0]" just presents a default selection of the first option, so it's not conflicting with any existing s.o.t

Reply:
sorry, not that part. The context part and how it plays with the other stuff (like setting the data in the reducer)

Reply:
Not sure what you mean, but i can tell you that this is only pulling data from context, not dispatching, so i'm not sure what the potential conflict would be - sorry i seem to be missing the point?

Reply:
Aw whoops, nope. I was confused.